<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>870</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Tafor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="mainTab">
      <property name="minimumSize">
       <size>
        <width>850</width>
        <height>0</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="recentTab">
       <attribute name="title">
        <string>RECENT</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QScrollArea" name="recentArea">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustToContents</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <widget class="QWidget" name="scrollContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>828</width>
             <height>544</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background:#fff</string>
           </property>
           <layout class="QVBoxLayout" name="recentLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <item>
             <widget class="QWidget" name="tips" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <layout class="QHBoxLayout" name="tipsLayout">
               <property name="leftMargin">
                <number>10</number>
               </property>
               <property name="topMargin">
                <number>20</number>
               </property>
               <property name="rightMargin">
                <number>10</number>
               </property>
               <property name="bottomMargin">
                <number>20</number>
               </property>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tafTab">
       <attribute name="title">
        <string>TAF</string>
       </attribute>
       <layout class="QVBoxLayout" name="tafLayout"/>
      </widget>
      <widget class="QWidget" name="metarTab">
       <attribute name="title">
        <string>METAR</string>
       </attribute>
       <layout class="QVBoxLayout" name="metarLayout"/>
      </widget>
      <widget class="QWidget" name="sigmetTab">
       <attribute name="title">
        <string notr="true">SIGMET</string>
       </attribute>
       <layout class="QVBoxLayout" name="sigmetLayout"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>870</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="issueMenu">
    <property name="title">
     <string>Issue</string>
    </property>
    <addaction name="tafAction"/>
    <addaction name="trendAction"/>
    <addaction name="sigmetAction"/>
    <addaction name="separator"/>
    <addaction name="quitAction"/>
   </widget>
   <widget class="QMenu" name="settingMenu">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="contractsMenu">
     <property name="title">
      <string>Contacts</string>
     </property>
     <addaction name="contractNo"/>
     <addaction name="separator"/>
    </widget>
    <addaction name="warnTafAction"/>
    <addaction name="contractsMenu"/>
    <addaction name="separator"/>
    <addaction name="settingAction"/>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="openDocsAction"/>
    <addaction name="reportIssueAction"/>
    <addaction name="checkUpgradeAction"/>
    <addaction name="separator"/>
    <addaction name="enterLicenseAction"/>
    <addaction name="removeLicenseAction"/>
    <addaction name="separator"/>
    <addaction name="aboutAction"/>
   </widget>
   <addaction name="issueMenu"/>
   <addaction name="settingMenu"/>
   <addaction name="helpMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="tafAction">
   <property name="text">
    <string>Terminal Aerodrome Forecast</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+F</string>
   </property>
  </action>
  <action name="trendAction">
   <property name="text">
    <string>Trend Forecast</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+T</string>
   </property>
  </action>
  <action name="sigmetAction">
   <property name="text">
    <string>Significant Meteorological Information</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+S</string>
   </property>
  </action>
  <action name="settingAction">
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+,</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="quitAction">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="contractNo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>None</string>
   </property>
  </action>
  <action name="reportIssueAction">
   <property name="text">
    <string>Report Issue</string>
   </property>
  </action>
  <action name="openDocsAction">
   <property name="text">
    <string>Documents</string>
   </property>
  </action>
  <action name="warnTafAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Alarm Sound</string>
   </property>
  </action>
  <action name="checkUpgradeAction">
   <property name="text">
    <string>Check for Updates</string>
   </property>
  </action>
  <action name="previousPageAction">
   <property name="text">
    <string>Previous Page</string>
   </property>
  </action>
  <action name="nextPageAction">
   <property name="text">
    <string>Next Page</string>
   </property>
  </action>
  <action name="removeLicenseAction">
   <property name="text">
    <string>Remove License</string>
   </property>
  </action>
  <action name="enterLicenseAction">
   <property name="text">
    <string>Enter License</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>quitAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>440</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
